src/database/models/ids.rs

```diff
@@ -102,9 +102,6 @@ pub struct TeamMemberId(pub i64);
 #[derive(Copy, Clone, Debug, Type)]
 #[sqlx(transparent)]
 pub struct ModId(pub i64);
-#[derive(Copy, Clone, Debug, Type)]
-#[sqlx(transparent)]
-pub struct StatusId(pub i64);
```

Comment:
This should be kept, but changed to contain an `i32`.  This is in the database models, so it should represent the id in the database.